core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -273,6 +273,104 @@ final RelMetadataFixture sql(String sql) {
                 / (DEPT_SIZE + EMP_SIZE)));
   }
 
+  // ----------------------------------------------------------------------
+  // Tests for getForeignKeys
+  // ----------------------------------------------------------------------
+
+  @Test void testForeignKeysAggregateCallEmpty() {
```

Comment:
What's the difference with the test right below this one?

Reply:
It‘s exactly same, i will fix it.